microsoft/iot-cardboard-js
# Conditions flyout form

### Summary of changes 🔍 
These changes should enable the Conditions flyout to be enabled so users can create/edit conditions in the Visual rule form. This should wrap up the overall functionality of Status and Alerts merge on the Builder side, however flighting is still in place until the viewer experience is updated.

TODO (in following PRs): 
- Remove flighting, will be done after viewer experience is updated.
- Add icon in Conditions list items
- Viewer experience

### Testing 🧪
#### Flight local storage key, to be able to see the functionality:
- Open developer tools of the browser and select application tab.
- Open localstorage variables
- Add this key: cardboard.feature.visualRulesFeature and set its value to true
- Refresh the page
OR -
- Open browser dev tools and run this in console: localStorage.setItem('cardboard.feature.visualRulesFeature',true);

#### To test:
- Open a scene and navigate to behaviors form
- Click on visual rules button
- Test out the UI for visual rules form. Suggested things to check out:
  - Edit a condition
  - Change the property type or expression and write new conditions based on those
  - Save changes and come back to review them
  - Click different conditions on the list to be able to see their values

### Checklist ✔️
